src/controllers/users.js

```diff
@@ -1,24 +1,32 @@
-const MongoClient = require('mongodb').MongoClient
-const config = require('../config')
+const MongoClient = require("mongodb").MongoClient;
```

Comment:
Dont thin Prettier is configured correctly - it's picked up on this file for double quotes and missign semi-colons, but not src/index.js, etc?